xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/internal/splitter/DocumentSplitterUtils.java

```diff
@@ -78,16 +78,17 @@ public static NamingCriterion getNamingCriterion(String namingCriterionId, Strin
     }
 
     /**
-     * Move artifacts (i.e. embedded images) from the original office document to a specific wiki document corresponding
-     * to a section. Only the artifacts from that section are moved.
+     * Add artifacts (i.e. embedded images) from the original office document to a specific wiki document corresponding
```

Comment:
This is misleading because I don't see any ``sectionDoc.add*`` call below (to justify that the artifacts are added to the section document). AFAICS the artifacts are not added (by this method) but returned, so why not call it ``getArtifacts`` or ``getSectionArtifacts``?

Reply:
That's true.  I changed the name to getSectionArtifacts.